packages/web/src/components/tracks-table/TracksTable.module.css

```diff
@@ -41,7 +41,11 @@
 
 .trackActionsContainer {
   display: inline-flex;
-  gap: 16px;
+  align-items: center;
+  justify-content: flex-end;
+  width: 100%;
+  gap: var(--harmony-spacing-l);
+  margin: 0 var(--harmony-spacing-l);
 }
```

Comment:
I say we just make this thing a `Flex`. I don't mind leaving stuff alone when it's just tweaking a value, but here we might as well migrate.

Reply:
True, I'm on board with that